README.md

```diff
@@ -90,17 +90,17 @@ see [devel/README.md](devel/README.md).
 ## File Structure
 
 ### Quick Reference
-| Directory | Description |
-| --------- | ----------- |
-| [`/config`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/config) | webpack configuration |
-| [`/devel`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/devel) | tools for local development |
-| [`/src`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/src) | source code |
-| [`/src/Components`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/src/Components) | source code split by individual components |
-| [`/src/test`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/src/test) | test utilities |
-| [`/src/test/mocks`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/src/test/mocks) | mock handlers and server config for MSW |
-| [`/src/store`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/src/store) | Redux store |
-| [`/src/api.js`](https://github.com/RedHatInsights/image-builder-frontend/blob/main/src/api.js) | API calls |
-| [`/src/store/apiSlice.js`](https://github.com/RedHatInsights/image-builder-frontend/blob/main/src/store/apiSlice.js) | RTK Query calls |
+| Directory                                                                                                            | Description                                |
+| ---------                                                                                                            | -----------                                |
+| [`/api`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/api)                                     | webpack configuration                      |
```

Comment:
Just one small suggestion, otherwise looks great to me!

```suggestion
| [`/api`](https://github.com/RedHatInsights/image-builder-frontend/tree/main/api)                                     | API config files                      |
```